pyramid_mailer/__init__.py

```diff
@@ -20,6 +20,15 @@ def includeme(config):
     settings = config.registry.settings
     prefix = settings.get('pyramid_mailer.prefix', 'mail.')
     mailer = mailer_factory_from_settings(settings, prefix=prefix)
+    set_mailer(config, mailer)
+
+
+def set_mailer(config, mailer):
+    """Set mailer in application config.
```

Comment:
Should this be named with an underscore (`_set_mailer`) to emphasize that it is not published API?